src/page/bank-payment.page.ts

```diff
@@ -7,7 +7,7 @@ export class BankPaymentPage {
     // this.confirmOrder = $('#cart_navigation > button > span');
     // I changed the selector because in the code block of #cart_navigation
     // is only one 'span', so whit those two thing we can identidy the element that we need.
-    this.confirmOrder = $('#cart_navigation span');
+    this.confirmOrder = element(by.cssContainingText('span', 'I confirm my order'));
```

Comment:
good job! also, other alternative that you could use was: `element(by.partialButtonText('I confirm my order'))`. Both options are fine :)